cert/openssl.cnf.old

```diff
@@ -0,0 +1,234 @@
+# SAS OpenSSL configuration file.
```

Comment:
No need to keep this file -- you can just delete it.

Are there other files in the sasCA directory that should be deleted as well?